aws/scalardl/example.tfvars

```diff
@@ -13,21 +13,17 @@ scalardl = {
   # green_image_tag              = "2.1.0"
   # green_image_name             = "ghcr.io/scalar-labs/scalar-ledger"
   # green_discoverable_by_envoy  = "false"
+  # container_env_file           = "scalardl_container.env"
+  # port                         = "50051"
+  # privileged_port              = "50052"
```

Comment:
Do we need these ports?

Reply:
Oh, right, these vars are not used anymore. Removed in 5f4105f. Thanks!